draft-ietf-gnap-core-protocol.md

```diff
@@ -204,29 +194,35 @@ role of the AS as defined by the protocol.
 In addition to the roles above, the protocol also involves several 
 elements that are acted upon by the roles throughout the process.
 
+Attribute
+: characteristics related to a subject.
+
 Access Token
-: A credential representing a set of access rights
-delegated to the client instance. The access token is created by the AS, consumed
-and verified by the RS, and issued to and carried by the client instance. The contents
-and format of the access token are opaque to the client.
+: a data artifact representing a set of rights and/or attributes.
+
+    Note: an access token can be first issued to an client instance (requiring authorization by the RO) and subsequently rotated.
 
 Grant
-: The process by which the client instance requests and is given delegated
-access to the RS by the AS through the authority of the RO.
+: (verb): to receive some attributes at a specific time and valid for a specific duration and/or to permit an instance of client software to exercise some set of delegated rights to access a protected resource (noun): the act of granting.
```

Comment:
This breaks the intended verb, which was "permit" for two actions of the client, to exercise and to receive. This should be instead:

```suggestion
: (verb): to permit an instance of client software to receive some attributes at a specific time and valid for a specific duration and/or to exercise some set of delegated rights to access a protected resource (noun): the act of granting.
```

Reply:
yes sorry, the problem with copy paste. Should be fixed now with latest commit.